.eslintrc.cjs

```diff
@@ -2,17 +2,20 @@ module.exports = {
   root: true,
   env: { browser: true, es2020: true },
   extends: [
-    'eslint:recommended',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:react-hooks/recommended',
+    "eslint:recommended",
```

Comment:
We could extend (ally, airBnB).
And maybe we could add some rules :
- max-lines-per-function (30) ? 
- typescript-eslint/no-explicit-any
- typescript-eslint/no-unused-vars